inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.dao.entity;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@Builder
+public class SortIdParamsEntity implements Serializable {
+    private Integer id;
+    private String taskName;
+    private String type;
+    private String groupId;
+    private String streamId;
+    private static final long serialVersionUID = 1L;
+}
```

Comment:
Here, the default value of serialVersionUID is "1L", which is inappropriate and should be random

Reply:
Auto generated by mybatis generator. 
I think Inlong must remain compatible with different release version, the value of serialVersionUID should be as easy as possible.